installing/installing_rhv/preparing-to-install-on-rhv.adoc

```diff
@@ -0,0 +1,26 @@
+[id="preparing-to-install-on-rhv"]
+= Preparing to install on RHV
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-rhv
+
+toc::[]
+
+[id="preparing-to-install-on-rhv-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="choosing-an-method-to-install-ocp-on-rhv"]
+== Choosing a method to install {product-title} on RHV
+
+You can install {product-title} on {rh-virtualization-first} by using one of the following deployment methods.
+
+* **xref:../../installing/installing_rhv/installing-rhv-default.adoc#installing-rhv-default[Installing a cluster quickly on RHV]**: You can quickly install {product-title} on {rh-virtualization} virtual machines that the {product-title} installation program provisions.
+
+* **xref:../../installing/installing_rhv/installing-rhv-customizations.adoc#installing-rhv-customizations[Installing a cluster on RHV with customizations]**: You can install a customized {product-title} cluster on installer-provisioned guests on {rh-virtualization}. The installation program allows for some customization to be applied at the installation stage. Many other customization options are available xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[post-installation].
+
+* **xref:../../installing/installing_rhv/installing-rhv-user-infra.adoc#installing-rhv-user-infra[Installing a cluster on RHV with user-provisioned infrastructure]**: You can install {product-title} on {rh-virtualization} virtual machines that you provision. You can use the provided Ansible playbooks to assist with the installation.
+
+* **xref:../../installing/installing_rhv/installing-rhv-restricted-network.adoc#installing-rhv-restricted-network[Installing a cluster on RHV in a restricted network]**: You can install {product-title} on {rh-virtualization} in a restricted or disconnected network, by creating an internal mirror of the installation release content. You can use this method to install a user-provisioned cluster that does not require an active internet connection to obtain the software components. You can also use this installation method to ensure that your clusters only use container images that satisfy your organizational controls on external content.
```

Comment:
I don't think you need the comma in "in a restricted or disconnected network, by creating an internal mirror "

other than that, looks good to me.

Reply:
@emarcusRH thank you for your review and for highlighting this. I have now resolved this suggestion.